Bottom: f39601900a1518a28293d31894f89f467213492f
Top:    7a70c6613840cb4d96e60831e25bd0c8c68c279b
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-02-24 07:12:40 -0300

Refresh of silence-warnings-and-improve-hashing.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index b7f1492..6466de9 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -283,7 +283,6 @@ address_rewriter (tree *in, int *walk_subtrees, void *arg)
   decl_addr_value value;
   value.decl = *in;
   decl_addr_value *found_value = ctx->address_map.find (&value);
-
   if (found_value != NULL)
     ;
   else if (DECL_IS_BUILTIN (*in) && DECL_NAMESPACE_SCOPE_P (*in)
